telethon/extensions/markdown.py

```diff
@@ -163,9 +163,12 @@ def unparse(text, entities, delimiters=None, url_fmt=None):
             text = text[:s] + delimiter + text[s:e] + delimiter + text[e:]
         elif isinstance(entity, MessageEntityTextUrl) and url_fmt:
             text = (
-                text[:s] +
-                add_surrogate(url_fmt.format(text[s:e], entity.url)) +
-                text[e:]
+                    text[:s] +
+                    add_surrogate(url_fmt.format(text[s:e], entity.url)) +
+                    text[e:]
             )
-
+        elif isinstance(entity, MessageEntityMentionName):
+            text = text[:entity.offset] + "[{}](tg://user?id={})".format(
```

Comment:
Shouldn't you use `s` and `e` for `offset` and `offset + length` respectively like everywhere else? It's shorter, more consistent, and probably faster (due to local variable vs. member access).